src/components/ArticlePage/Toc.js

```diff
@@ -11,10 +11,9 @@ const TocLi = styled(Li).attrs({
   // Side navigation hidden on small viewport
 
   @media (min-width: ${baselineBreakpoint}px) {
-    line-height: 32px;
-
-    // Offset baseline
-    top: 2px;
+    line-height: 24px;
+    padding-bottom: ${baselineLarge}px;
+    font-size: 15px;
```

Comment:
This is now out of baseline alignment. Add the offset:

```suggestion
    font-size: 15px;
    
    // Offset baseline
    top: -1px;
```